projects/js-packages/social-previews/package.json

```diff
@@ -1,82 +1,69 @@
 {
 	"name": "@automattic/social-previews",
-	"version": "2.1.0-beta.13",
+	"version": "2.0.1",
 	"description": "A suite of components to generate previews for a post for both social and search engines.",
-	"main": "dist/cjs/index.js",
-	"module": "dist/esm/index.js",
-	"calypso:src": "src/index.ts",
-	"exports": {
-		".": {
-			"calypso:src": "./src/index.ts",
-			"types": "./dist/types/index.d.ts",
-			"import": "./dist/esm/index.js",
-			"require": "./dist/cjs/index.js"
-		}
-	},
-	"sideEffects": [
-		"*.css",
-		"*.scss"
-	],
 	"keywords": [
 		"wordpress",
 		"social",
 		"seo",
 		"search"
 	],
-	"author": "Automattic Inc.",
-	"homepage": "https://github.com/Automattic/wp-calypso",
-	"license": "GPL-2.0-or-later",
+	"homepage": "https://github.com/Automattic/jetpack/tree/HEAD/projects/js-packages/social-previews/#readme",
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/labels/[JS Package] Social Previews"
+	},
 	"repository": {
 		"type": "git",
-		"url": "git+https://github.com/Automattic/wp-calypso.git",
-		"directory": "packages/social-previews"
-	},
-	"publishConfig": {
-		"access": "public"
+		"url": "https://github.com/Automattic/jetpack.git",
+		"directory": "projects/js-packages/social-previews"
 	},
-	"bugs": "https://github.com/Automattic/wp-calypso/issues",
-	"files": [
-		"dist",
-		"src",
-		"CHANGELOG.md"
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic",
+	"sideEffects": [
+		"*.css",
+		"*.scss"
 	],
-	"types": "dist/types",
+	"exports": {
+		".": {
+			"jetpack:src": "./src/index.ts",
+			"import": "./dist/index.mjs",
+			"require": "./dist/index.js"
+		},
+		"./style.css": "./dist/index.css"
+	},
+	"main": "dist/index.js",
+	"module": "dist/index.mjs",
 	"scripts": {
-		"clean": "tsc --build ./tsconfig.json ./tsconfig-cjs.json --clean && rm -rf dist",
-		"build": "tsc --build ./tsconfig.json ./tsconfig-cjs.json && copy-assets",
-		"prepare": "yarn run build",
-		"prepack": "yarn run clean && yarn run build",
-		"watch": "tsc --build ./tsconfig.json --watch"
+		"build": "pnpm run build:prod",
+		"build:prod": "tsup",
+		"test": "jest --config=tests/jest.config.cjs",
+		"test-coverage": "pnpm run test --coverage",
+		"typecheck": "tsc --noEmit"
 	},
 	"dependencies": {
-		"@emotion/react": "^11.11.1",
-		"@wordpress/components": "^30.9.0",
-		"@wordpress/element": "^6.23.0",
-		"@wordpress/i18n": "^5.23.0",
-		"clsx": "^2.1.1",
-		"prop-types": "^15.8.1",
-		"tslib": "^2.8.1"
+		"@wordpress/components": "30.9.0",
+		"@wordpress/element": "6.36.0",
+		"@wordpress/i18n": "6.9.0",
+		"clsx": "2.1.1",
+		"react": "18.3.1",
+		"react-dom": "18.3.1"
```

Comment:
Why did you make `react` and `react-dom` dependencies instead of leaving them as peer dependencies?

Reply:
Good idea. Thanks

Moved them to peer and dev deps in https://github.com/Automattic/jetpack/pull/46315/commits/6438f7249d842ac9efdc03f64e0b1809b2fe3153